metadata-ingestion/setup.py

```diff
@@ -61,6 +61,7 @@ def get_long_description():
     "jsonschema<=4.17.3; python_version < '3.8'",
     "jsonschema; python_version >= '3.8'",
     "ruamel.yaml",
+    "memray"
```

Comment:
this is fine for testing using the vercel url, but I don't want to have memray be a required dependency when it's only used for an opt-in feature

Reply:
Is there a way to opt-in into dependencies?

Reply:
yep you can add it as an "extra"

that said, our ingestion UI doesn't support requesting extras yet (even though acryl-executor does), so the current workaround is to manually install it into the venv

Reply:
Moved memray to a "debug" subpackage. However won't this cause issues since we unconditionally import and use `memray` in the `ingest_cli.py` ?

Reply:
yep - we should also move the `import` statement inside the if statement